### PR TITLE
Add Phototherapy e2e tests

### PR DESCRIPTION
**Pull request by DanielKag**
Adds e2e tests to phototherapy app.

**The plan is (in future PRs):**
- Make all tests pass
- Refactor logic code (add Unit tests)
- Migrate to a cutting-edge UI framework (React? Vue?)
- Remove the e2es, expect of some sanity tests

### CODE REVIEW

File: phototherapy/js/phototherapy.spec1.js
Comment (DanielKag) [inline phototherapy/js/phototherapy.spec1.js:1]: @talavinari @yonatankatz 
This test is angularJs specific, and it breaks on master.
Do we still need it or can I remove it?

File: phototherapy/js/phototherapy.spec.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// Do we still need this file?
```
Comment (DanielKag) [inline phototherapy/js/phototherapy.spec.js]: @talavinari @yonatankatz 
This test is angularJs specific, and it breaks on master.
Do we still need it or can I remove it?

Comment (DanielKag) [reply phototherapy/js/phototherapy.spec.js]: And do we need `karma` for testing?
`jest` is much more popular and well maintained, `karma` is kinda dead :/

Comment (talavinari) [reply phototherapy/js/phototherapy.spec.js]: totally don't need it. it is our first try to kick off the E2E testing. we can safely delete the spec file and the karma package

Comment (DanielKag) [reply phototherapy/js/phototherapy.spec.js]: removed `karma`

**Pull request merged**